smart-contracts/contracts/mixins/MixinRefunds.sol

```diff
@@ -108,6 +108,7 @@ contract MixinRefunds is
 
     // make future reccuring transactions impossible
     _originalDurations[_tokenId] = 0;
+    _originalPrices[_tokenId] = 0;
```

Comment:
Oh, now that I think about it I wonder what happens on renewals for a lock that has free keys?

Reply:
yes why I using with duration together is safer